src/avatar.scss

```diff
@@ -40,7 +40,7 @@ $block: #{$fd-namespace}-avatar;
     [class*="#{$block}"] {
       @include fd-reset();
 
-      color: var(--sapContent_ContrastTextColor);
+      color: var(--fdAvatar_DefaultColor);
       background-color: $fd-avatar-background-color;
```

Comment:
`$fd-avatar-background-color` needs a new variable - it should be `--sapAccentColor6` in light + dark, `--sapBackgroundColor` in hcw/hcb.